packages/office-ui-fabric-react/src/components/HoverCard/CardCallout/CardCallout.tsx

```diff
@@ -5,6 +5,7 @@ import { Callout } from '../../../Callout';
 import { DirectionalHint } from '../../../common/DirectionalHint';
 import { FocusTrapZone } from '../../../FocusTrapZone';
 import { IBaseCardProps } from '../BaseCard.types';
+import { FocusTrapCallout, ICalloutProps } from '../../Callout';
 
 export interface ICardCalloutProps extends IBaseCardProps<{}, {}, {}> {
```

Comment:
I just realized that whatever I said above about FocusTrapCallout applies to this very component itself too, although at least this is a stateless component (though not explicitly typed as such).

Can we document these props?